From 1486e63a7180090a20c8f047a1a5fe74362489a0 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.liu@freescale.com>
Date: Tue, 24 Jul 2012 18:10:23 +0800
Subject: ENGR00182061-4 MXC v4l2 capture:Remove inappropriate msleep code

msleep() after camera power down should not be called in mxc
v4l2 capture core code. Instead, this should be handled by
camera power down function.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 drivers/media/video/mxc/capture/mxc_v4l2_capture.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/media/video/mxc/capture/mxc_v4l2_capture.c b/drivers/media/video/mxc/capture/mxc_v4l2_capture.c
index e8643ab..ab41d9c 100644
--- a/drivers/media/video/mxc/capture/mxc_v4l2_capture.c
+++ b/drivers/media/video/mxc/capture/mxc_v4l2_capture.c
@@ -1681,7 +1681,6 @@ static int mxc_v4l_open(struct file *file)
 		ipu_csi_enable_mclk_if(cam->ipu, CSI_MCLK_I2C, cam->mclk_source,
 				       true, true);
 		vidioc_int_s_power(cam->sensor, 1);
-		msleep(1);
 		vidioc_int_init(cam->sensor);
 		vidioc_int_dev_init(cam->sensor);
 	}
-- 
1.7.9.5

